packages/plugin-ext/src/plugin/languages.ts

```diff
@@ -446,13 +449,26 @@ export class LanguagesExtImpl implements LanguagesExt {
         metadata?: theia.CodeActionProviderMetadata
     ): theia.Disposable {
         const callId = this.addNewAdapter(new CodeActionAdapter(provider, this.documents, this.diagnostics, pluginModel ? pluginModel.id : '', this.commands));
+
+        let documentation: CodeActionProviderDocumentation | undefined;
+        let disposables: DisposableCollection | undefined;
+        if (metadata && metadata.documentation) {
+            disposables = new DisposableCollection();
+            documentation = metadata.documentation.map(doc => ({
+                kind: doc.kind.value!,
```

Comment:
@lucas-koehler I believe that [`value`](https://code.visualstudio.com/api/references/vscode-api#CodeActionKind) is non-optional in vscode, should we align?

- [vscode@master](https://github.com/microsoft/vscode/blob/25ee7d964c45b42d42c7afdc8bf619550cb72a17/src/vscode-dts/vscode.d.ts#L2216-L2219)
- [vscode@1.53.2](https://github.com/microsoft/vscode/blob/622cb03f7e070a9670c94bae1a45d78d7181fbd4/src/vs/vscode.d.ts#L2127-L2130)